bin/samoa-flink.properties

```diff
@@ -0,0 +1,35 @@
+###
+# #%L
+# SAMOA
+# %%
+# Copyright (C) 2013 Yahoo! Inc.
```

Comment:
The copyright info is outdated.
I think this has recently been fixed in the pom file.